server/routes/send.js

```diff
@@ -25,31 +26,30 @@ module.exports = ({ licenceService, prisonerService, notificationService, audit
         res.locals.token
       )
 
-      const notifications = await notificationService.getNotificationData({
-        prisoner: res.locals.prisoner,
-        token: res.locals.token,
-        notificationType: transition.notificationType,
-        submissionTarget,
-        bookingId,
-        sendingUser: req.user,
-      })
-
-      await Promise.all([
-        licenceService.markForHandover(bookingId, transition.type),
-        notificationService.notify({
-          user: req.user.username,
-          type: transition.notificationType,
-          bookingId,
-          notifications,
-        }),
-      ])
+      await licenceService.markForHandover(bookingId, transition.type)
 
       if (transition.type === 'dmToCaReturn') {
         await licenceService.removeDecision(bookingId, res.locals.licence)
       }
 
       auditEvent(req.user.username, bookingId, transition.type, submissionTarget)
 
+      try {
```

Comment:
I'd say instead of having the try catch here perhaps it's better to have it inside the `sendNotifications` method have that deal with the error silently.. you could even test that if you wanted to?

Reply:
Good point. Will fix